autorest/azure/environments.go

```diff
@@ -135,6 +137,7 @@ var (
 			Datalake:            NotAvailable,
 			Batch:               "https://batch.core.usgovcloudapi.net/",
 			OperationalInsights: "https://api.loganalytics.us",
+			Storage:             "https://storage.azure.com/",
```

Comment:
so these are the same in all env ?

Reply:
Correct, see https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/services-support-managed-identities#azure-storage-blobs-and-queues